frontend/src/home/Home.tsx

```diff
@@ -338,20 +293,20 @@ class HomeComponent extends React.Component<Props, HomeState> {
     // if drag is coming from the sidebar
     if (isNaN(Number(source.droppableId))) {
       addCourseFromSidebar(
-        this.props.schedule,
+        this.props.activePlan!.schedule,
```

Comment:
In order to be able to add a course from sidebar, there must be an active plan, so it's okay to assert here.